### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by uncle9x9**

### CODE REVIEW

**Comment by WinsonLiGeneration** [comment]
@winsonliwh , I'm ok with this. Please review the code.

**Review by WinsonLiGeneration — COMMENTED** [review]
@winsonliwh , I'm ok with this. Pls approve.

**Review by winsonliwh — APPROVED** [review]
@WinsonLiGeneration , I approved. Pls merge to main.

**Comment by WinsonLiGeneration** [comment]
@uncle9x9 , I need to check something before merge.

File: index.html [hunk 1]
```diff
@@ -1,2 +1,3 @@
 index.html
 Swithed to github-uncle9x9 which is fetched from origin
+Switched to newFeature1 branch and added line 3 into index.html
```
Comment (WinsonLiGeneration) [inline index.html:3]: @uncle9x9 @WinsonLiGeneration , I need to check something before merge.

**Review by WinsonLiGeneration — APPROVED** [review]
@uncle9x9 @winsonliwh , it is ok after checking.

File: index.html (hunk 1 above)
Comment (WinsonLiGeneration) [reply index.html:3]: After checking, it is ok.

**Review by winsonliwh — APPROVED** [review]
@WinsonLiGeneration , after your checking is good. And I agree the result of checking.

**Pull request merged**